int32 light_mode  
float64 battery_voltage
uint8 motor_num
float64 linear
float64 angular
float64 motor_current
float64 motor_rpm
uint16 motor_pulses
float64 motor_temperature
float64 driver_voltage
uint8 driver_state
float64 driver_temperature
